src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/README.md

```diff
@@ -24,6 +24,7 @@ In order to publish the extension, you will need to have access to the `ms.dotne
 
 This extension bundles the debugging proxy assets that are needed inside the `BlazorDebugProxy` directory. These assets are not committed to repository so they will need to be included as part of the publish process.
 
-1. Download the desired version of the [Microsoft.AspNetCore.Components.WebAssembly.DevServer](https://www.nuget.org/packages/Microsoft.AspNetCore.Components.WebAssembly.DevServer/) package.
-2. Copy the contents of `tools/BlazorDebugProxy` into `BlazorDebugProxy/` in this directory.
-3. Package and publish the extension.
\ No newline at end of file
+1. Generate a personal access token per the instructions in [the VS Code publishing guide](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page).
```

Comment:
Holy cow I had no idea. Does this mean that anyone can publish under the ms-dotnettools publisher as long as they provide a PAT?